src/Test/Assert.ss

```diff
@@ -9,20 +9,14 @@
           (if (not success) 
             (raise-continuable
               (condition
-                (make-serious-condition)
                 (make-message-condition message))))))))
 
   (define checkThrows
     (lambda (fn)
       (lambda ()
-        (with-exception-handler
-          (lambda (e)
-            (unless (serious-condition? e)
-              (raise
-                (condition
-                  (make-violation)
-                  (make-message-condition "Threw something other than a serious condition"))))
-            #t)
-          (lambda () (eq? (fn 'unit) #t))))))
+        (call/cc (lambda (k)
+          (with-exception-handler
+            (lambda (e) (k #t))
+            (lambda () (eq? (fn 'unit) #t))))))))
```

Comment:
This will be a false positive if `fn` is `Unit -> Boolean`. You can use `(and (fn 'unit) #f)` instead.

Reply:
Or maybe even `begin`, that would be more explicit.

Reply:
ping @anttih

Reply:
Sorry been focusing on other things. Would you like to submit a PR for this? I can also look at this later.

Reply:
Done